include/powsybl/iidm/VoltageLevelViews.hpp

```diff
@@ -161,9 +163,9 @@ class NodeBreakerView {
 
     virtual SwitchAdder newSwitch() = 0;
 
-    virtual void removeSwitch(const std::string& switchId) = 0;
+    virtual void removeInternalConnections(unsigned long node1, unsigned long node2) = 0;
 
-    virtual NodeBreakerView& setNodeCount(unsigned long nodeCount) = 0;
+    virtual void removeSwitch(const std::string& switchId) = 0;
```

Comment:
You forgot to remove getNodeCount(): if you want to manage backward compatibility, add a default behavior and make this method not virtual

Reply:
Ok removed. To be documented in V1.2.0 release note